Minor improvements related to readability

sagar-jadhav opened the pull request:
- Fixed grammar of few sentences

sagar-jadhav commented:
@viveksyngh could you please review this Pull Request? All the changes are minor doc updates.

viveksyngh commented on README.md:
````diff
@@ -12,20 +12,29 @@ Stable binaries are also available on the [releases](https://github.com/viveksyn
 ## For Linux:
 ```
 curl -Lo ./kube-sync https://github.com/viveksyngh/kube-sync/releases/download/v0.1.0/kube-sync-$(uname)-amd64
+```
````
What is the benefit on putting them in separate code block ? The reason they were put in same code block because then can be copied and pasted in one go.

sagar-jadhav replied:
I agree that If we put all commands in one block then it can be copied and pasted in one go.

But I suggested this change from a user perspective, When I followed the documentation I realize that executing command will be more convenient If executed one by one. Hence I put each command in a separate block. 

Let me know If you still want me to revert this change.

viveksyngh replied:
I would like to keep this same as it is for now. May be later we can revisit this after more feedback.

viveksyngh commented on README.md:
````diff
@@ -12,20 +12,29 @@ Stable binaries are also available on the [releases](https://github.com/viveksyn
 ## On macOS via Bash:
 ```
 curl -Lo ./kube-sync https://github.com/viveksyngh/kube-sync/releases/download/v0.1.0/kube-sync-darwin-amd64
+```
````
Same as above

sagar-jadhav replied:
done

viveksyngh commented on README.md:
````diff
@@ -12,20 +12,29 @@ Stable binaries are also available on the [releases](https://github.com/viveksyn
 ```
 curl.exe -Lo kube-sync-windows-amd64.exe https://github.com/viveksyngh/kube-sync/releases/download/v0.1.0/kube-sync-windows-amd64
-
+```
````
Same as above.

sagar-jadhav replied:
done

viveksyngh commented on README.md:
```diff
@@ -1,9 +1,9 @@
 # Motivation
 
-While working with kubernetes, all of us might have encountered a situation where applications deployed in a new namespace were failing because of missing image pull secret, which we forgot to create in the new namespace. This tool help us in copying such secrets or configmaps from already existing namespace to a new namespace.
+While working with Kubernetes, all of us might have encountered a situation where applications deployed in a new namespace were failing because of missing image pull secret, which we forgot to create in the new namespace. This tool helps us in copying such secrets or ConfigMaps from an already existing namespace to a new namespace.
```
Thanks for updating this.

viveksyngh commented on README.md:
```diff
@@ -48,8 +57,8 @@ COMMANDS:
    --kubeconfig value                    path to the kubeconfig file to use for CLI requests (default: "~/.kube/config") [$KUBECONFIG]
-   --namespace value, -n value           namespace of the source resource (default: "default")
-   --target-namespace value, --tn value  target namespace in which resource will be copied (default: "default")
+   --namespace value, -n value           The namespace of the source resource (default: "default")
```
I am not 100% sure that we need to prefix these two line with `The`

sagar-jadhav replied:
ok in that case I'll revert this change.

viveksyngh commented on cmd/validator.go:
```diff
@@ -8,11 +8,11 @@ import (
 func Validate(ctx *cli.Context) error {
 	if ctx.Args().Len() < 1 {
-		return fmt.Errorf("resource name is missing")
+		return fmt.Errorf("The resource name is missing")
```
This `the` is not required. We can remove this.

sagar-jadhav replied:
ok i'll revert this change.

viveksyngh commented on main.go:
```diff
@@ -92,14 +92,14 @@ func main() {
 				Aliases:     []string{"n"},
 				Value:       "default",
-				Usage:       "namespace of the source resource",
+				Usage:       "The namespace of the source resource",
```
This `The` can be remove in my opinion.

sagar-jadhav replied:
ok i'll revert this change.

viveksyngh commented on main.go:
```diff
@@ -92,14 +92,14 @@ func main() {
 				Aliases:     []string{"tn"},
 				Value:       "default",
-				Usage:       "target namespace in which resource will be copied",
+				Usage:       "The target namespace where resource will be copied",
```
This one as well.

sagar-jadhav replied:
ok i'll revert this change.

viveksyngh reviewed (changes requested):
Left some comments. Please address them and also squash the commits in one. I can still see 2 commits.

sagar-jadhav commented:
> Left some comments. Please address them and also squash the commits in one. I can still see 2 commits.

@viveksyngh I reverted few of the changes. The other commit is a merge commit. I tried to squash all three commits but as merge commit is in between the two other commit Git is not allowing me to squash.  For more details you can refer [here](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits). 

Regarding code block I have already shared my opinion please have a look and let me know what to do for that. Thanks

viveksyngh commented:
@sagar-jadhav Let's revert the code block changes. Somehow we need to squash commits into one. It wouldn't be good idea to merge multiple commits for a simple task of updating readme.

sagar-jadhav commented:
> @sagar-jadhav Let's revert the code block changes. Somehow we need to squash commits into one. It wouldn't be good idea to merge multiple commits for a simple task of updating readme.

@viveksyngh As you suggested I have reverted the commits and also squashed all the commits. Let me know If any additional changes required.

viveksyngh reviewed (approved):
LGTM
